### ISSUE TITLE
Add JList with ATF protocols from settings when creating ATF template

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#155]
I did a multiselection JList that loads a list of ATF protocols from the settings, but in didn't work out in the GUI plus Eleanor never asked explicitly, so I stopped working on it because we don't have much time.
It'd be nice to do this later on.

**Comment by raquel-ucl** [comment]
Jake has asked for something related to this, so perhaps we could make it higher priority. 

This is Jake's email:

> Hello Raquel,
>     
> My colleagues and I have recently started a new project on Oracc and have been creating the 
> transliterations of new texts via Nammu to great success.  I hope I can bother you with one question 
> that we have though:  Is there some way to make the template for creating a new text add 
> #atf: use math to the protocol lines at the start of each text?
>     
> Jake

Since I made an attempt to work on this some time ago, the `settings.yml` file already includes a list of protocols:

```
protocols: [math, legacy, lexical, mylines]
```

I have modified the code so it takes this list into account and adds the protocols to the template, with no changes on the GUI. This way, users like Jake can open the `settings.yaml` file and add/remove the protocols they want from the list. However if they mess up the settings file, other things might break - hence I'd like to make it only editable from the settings menu on Nammu.

This last option takes about 2-3 days of work. Should we give it more priority @EleanorRobson ? At the moment is descoped and marked as low priority.

I've made some changes and a PR for Stuart to review and will try to make a release tomorrow for Jake if all is well.
